app/templates/components/code-theme-selector.hbs

```diff
@@ -0,0 +1 @@
+{{yield}}
```

Comment:
It's usually auto-generated if you're generating a component, so there might be.